### PR TITLE
RD-3731: Hidded help popup when none of the help informations should …

### PR DESCRIPTION
**Pull request by NorbertAmartus**
## Description

Changes consists of returning `null`, instead of `<div />` , when none of the HelpProperties should be shown.

### Before 
Bez tytułu

### After 
localhost_4000_console_page_services

## Checklist

- [x] My code follows the UI Code Style.
- [x] I verified that all tests and checks have passed.
- [x] I verified if that change requires a documentation update.
- [x] I added proper labels to this PR.

## Tests

 

[System tests](https://jenkins.cloudify.co/view/UI%20Health%20View/job/Stage-UI-System-Test/1739/) has been executed
Additionally, I've run `deployments_view_spec.ts` cypress test locally

## Documentation

N/A

### CODE REVIEW

**Comment by NorbertAmartus** [comment]
Just a note regarding the [ongoing refactors](https://cloudifysource.atlassian.net/wiki/spaces/RD/pages/2148073473/Ongoing+Refactors+in+Cloudify+UI):

Regarding migrating files to ts (upon which I've been working on), [file](https://github.com/cloudify-cosmo/cloudify-stage/blob/c2c933c0563a1f9822a037dc6c330c973c55d8b3/widgets/common/src/InputsUtils.tsx) (which was modified by me) is large, that's why I didn't migrate it

File: widgets/common/src/InputsUtils.tsx [hunk 1]
```diff
@@ -197,11 +203,11 @@ class InputsUtils {
                 />
                 <HelpProperty
                     name={!_.isUndefined(defaultValue) ? 'Default Value' : 'Example'}
-                    show={!_.isUndefined(defaultValue) || !_.isUndefined(dataType)}
+                    show={showDefaultValue}
                     value={<ParameterValue value={example} />}
                 />
```
Comment (qooban) [inline widgets/common/src/InputsUtils.tsx:208]: Optional: 

I'm wondering what should be correct names here. This component shows either a default value of an input or an example - object shape (created based on the passed `dataType`). In L206 we use `showDefaultValue` name, in the other (L207) we use `example` ;-)
Maybe (thus optional) something to think about and fix, but not a blocker I suspect.

**Review by qooban — APPROVED** [review]
Looks good! 👍🏻 

Added just one optional comment.

File: widgets/common/src/InputsUtils.tsx (hunk 1 above)
Comment (NorbertAmartus) [reply widgets/common/src/InputsUtils.tsx:208]: Hmm, then I think that we may stick into the `showExample` variable as the `default value` may be too considered as an example :)

Please let me know, what do you think about it 😉

Comment (qooban) [reply widgets/common/src/InputsUtils.tsx:208]: OK, let's go with `showExample` then 👍🏻

Comment (NorbertAmartus) [reply widgets/common/src/InputsUtils.tsx]: Done 😉

**Review by qooban — APPROVED** [review]
Looks good 👍🏻

**Pull request merged**